python/foglamp/services/core/routes.py

```diff
@@ -71,6 +72,9 @@ def setup(app):
     app.router.add_route('DELETE', '/foglamp/backup/{backup_id}', backup_restore.delete_backup)
     app.router.add_route('PUT', '/foglamp/backup/{backup_id}/restore', backup_restore.restore_backup)
 
+    # Package Update on demand
+    app.router.add_route('PUT', '/foglamp/update', update.update_package)
```

Comment:
'/foglamp/snap/update', `update_snap.update_package` or `snap_package.update`

Reply:
Again, we don't want to specialise here with snap/yum/apt etc. Just be generic.